appIcons.js

```diff
@@ -231,14 +232,32 @@ const MyAppIcon = new Lang.Class({
         let event = Clutter.get_current_event();
         let modifiers = event ? event.get_state() : 0;
         let openNewWindow = modifiers & Clutter.ModifierType.CONTROL_MASK &&
-                            this.app.state == Shell.AppState.RUNNING ||
-                            button && button == 2;
```

Comment:
I removed `button && button == 2` as the behavior might not be "openNewWindow" anymore. Even if it is, the "Launch new instance" will open a new window now anyway..